### ISSUE TITLE
Cannot find namespace 'google'

### ISSUE THREAD

**Issue by nicolashimmelmann** [inouiw/capacitor-native-google-one-tap-signin#13]
Hi,

first of all, this seems like a really great plugin. Nice work!

When I add the plugin to my Angular Ionic project, I get the error "Cannot find namespace 'google'".

```
[ng] Error: node_modules/capacitor-native-google-one-tap-signin/dist/esm/GoogleOneTapAuth.d.ts:14:110 - error TS2503: Cannot find namespace 'google'.
[ng] 
[ng] 14     renderSignInButton(parentElementId: string, options: RenderSignInButtonOptions, gsiButtonConfiguration?: google.GsiButtonConfiguration): Promise<SuccessSignInResult>;
[ng]                                                                                                                 ~~~~~~
[ng] 
[ng] 
[ng] Error: node_modules/capacitor-native-google-one-tap-signin/dist/esm/GoogleOneTapAuth.d.ts:15:112 - error TS2503: Cannot find namespace 'google'.
[ng] 
[ng] 15     _doRenderSignInButton(parentElementId: string, options: RenderSignInButtonOptions, gsiButtonConfiguration: google.GsiButtonConfiguration | undefined, androidAndIosButtonElem: HTMLElement): Promise<SuccessSignInResult>;
[ng]                                                                                                                   ~~~~~~
[ng] 
[ng] 
[ng] Error: node_modules/capacitor-native-google-one-tap-signin/dist/esm/definitions.d.ts:139:110 - error TS2503: Cannot find namespace 'google'.
[ng] 
[ng] 139     renderSignInButton(parentElementId: string, options: RenderSignInButtonOptions, gsiButtonConfiguration?: google.GsiButtonConfiguration): Promise<SuccessSignInResult>;
[ng]
```

What helps is adding the package `@types/google-one-tap` you are using in the plugin to my dev dependencies and adding `/// <reference types="@types/google-one-tap" />` to `node_modules/capacitor-native-google-one-tap-signin/dist/esm/definitions.d.ts`.

Regards
Nicolas

**Comment by inouiw** [comment]
Hi Nicolas,
thanks for the issue.
I wanted to re-export the google types some time ago but then typescript did not support it. With TypeScript 5 it is now possible to export all types from an imported module. See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-0.html#support-for-export-type-

Do you want to create a pull request to fix the problem?
My preference would be to first try to upgrade to TypeScript 5, then use the `export type * as google from ...` syntax to export the types.

David

**Comment by inouiw** [comment]
@nicolashimmelmann thank you for the fix!!
I merged it and created a new version 5.1.2
